doc/api/child_process.md

```diff
@@ -299,6 +299,9 @@ output on this fd is expected to be line delimited JSON objects.
 *Note: Unlike the fork(2) POSIX system call, `child_process.fork()` does
 not clone the current process.*
 
+*Note*: Support for non-Array values of the `.stdio` option was added in
```

Comment:
Don't we usually put something in the API docs when we make breaking API changes? I'll check around to see what we've done in the recent past.